tikv/raftstore/coprocessor.go

```diff
@@ -1,11 +1,17 @@
 package raftstore
 
 import (
+	"bytes"
+	"fmt"
 	"github.com/coocood/badger"
 	"github.com/ngaut/log"
+	"github.com/ngaut/unistore/tikv"
```

Comment:
importing this package will cause dependency loop in the future, we need to move the sharding key function to `mvcc` package.

Reply:
yes, I would fix this.